draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1242,10 +1242,11 @@
           <section title="Carrying Identity Assertions">
             <t>
               Once an IdP has generated an assertion, it is attached to the SDP
-              message. This is done by adding a new identity attribute to the
-              SDP. The sole contents of this value are a <xref
-              target="RFC4648">base-64 encoded</xref> identity assertion.  For
-              example:
+              offer/answer message. This is done by adding a new identity
+              attribute to the SDP; the offer includes the caller's assertion,
+              the answer includes the callee's assertino. The sole contents of
```

Comment:
Typo: Assertion
There is a problem with identifying the caller and callee: they aren't always in the same offer/answer roles. I would avoid mentioning that (drop the bit after the semicolon).